components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -191,6 +191,12 @@ func (backend *ESClient) setDailyLatestToFalse(ctx context.Context, nodeId strin
 	return errors.Wrap(err, "setDailyLatestToFalse")
 }
 
+func (backend *ESClient) DeleteProjectTag(ctx context.Context,
+	projectTagToBeDelete string) ([]string, error) {
+	// TODO implement
```

Comment:
This will be completed with this issue https://app.zenhub.com/workspaces/automate-auth-5d1c1073e0945f63bfe3195b/issues/chef/automate/2388